cloudos_cli/__main__.py

```diff
@@ -2478,6 +2481,113 @@ def move_files(ctx, source_path, destination_path, apikey, cloudos_url, workspac
         click.echo(f"[ERROR] Move operation failed: {str(e)}", err=True)
         sys.exit(1)
 
+@datasets.command(name="rename")
+@click.argument("source_path", required=True)
+@click.argument("new_name", required=True)
+@click.option('-k', '--apikey', required=True, help='Your CloudOS API key.')
+@click.option('-c', '--cloudos-url', default=CLOUDOS_URL, required=True, help='The CloudOS URL.')
+@click.option('--workspace-id', required=True, help='The CloudOS workspace ID.')
+@click.option('--project-name', required=True, help='The project name.')
+@click.option('--disable-ssl-verification', is_flag=True, help='Disable SSL certificate verification.')
+@click.option('--ssl-cert', help='Path to your SSL certificate file.')
+@click.option('--profile', default=None, help='Profile to use from the config file.')
+@click.pass_context
+def renaming_item(ctx, source_path, new_name, apikey, cloudos_url,
+                    workspace_id, project_name,
+                    disable_ssl_verification, ssl_cert, profile):
+    """
+    Rename a file or folder in a CloudOS project.
+
+    SOURCE_PATH must be a full path like 'Data/folderA/old_name.txt'
+    NEW_NAME is the new name to assign.
+    """
+    if not source_path.strip("/").startswith("Data/"):
+        click.echo("[ERROR] SOURCE_PATH must start with  'Data/'.", err=True)
```

Comment:
```suggestion
        click.echo("[ERROR] SOURCE_PATH must start with 'Data/', pointing to a file/folder in that dataset.", err=True)
```
This is because if the user adds --source-path `Data/` it will trigger this error but the message is confusing, as it actually starts with `Data/`.